docs/modules/clients/pages/java.adoc

```diff
@@ -67,23 +67,75 @@ client.shutdown();
 
 The above code line releases all the used resources and closes connections to the cluster.
 
-=== Java Client Operation Modes
+=== Java Cluster Routing Modes
 
-The client has two operation modes because of the distributed nature of the data and cluster.
+The Java cluster routing mode specifies how the client connects to the cluster.
 
-**Smart Client**: In the smart mode, the clients connect to each cluster member.
+NOTE: In previous releases, this functionality was known as the client operation mode and could be configured as smart or unisocket. 
+If the cluster routing mode is not configured in your client, the configured client operation mode is used.
+
+The mode that you use depends on your network requirements due to the distributed nature of the data and cluster.
+
+In all modes, the following information is provided to the client on initial connection:
+
+* Cluster version
+* Member details, including instance name, IP or UUID, partition group information, CP membership status, and type ({enterprise-product-name} only)
+* Partition group information ({enterprise-product-name} only)
+* CP group leader information ({enterprise-product-name} with `Advanced CP` enabled only) 
+
+The client is updated whenever the cluster version, partition groups, or CP group leader changes.
+
+From {enterprise-product-name} 5.5, you can use any of the following cluster routing modes:
+
+* Legacy **Smart** and **Unisocket** client operation modes. For information on these modes and their configuration, select **5.4** from the version picker at the top of the navigation pane. Ensure that the cluster routing mode is not configured to use the configured client operation mode
```

Comment:
My original thinking was for people who already had it configured (a) to see how they can update their configuration if preferred, and (b) in case they wanted to change their current configuration/add a client with the legacy options 

However, I have no strong feelings on it, so long as the information is there, they should be able to find it ;-)